parsl.sh

```diff
@@ -1,15 +1,19 @@
 #!/bin/bash
 
-# Test for integration of Parsl and Work Queue starting from Conda install.
+# Test for integration of Parsl with Work Queue and TaskVine starting from Conda install.
 
 # Fix for local environment at ND: unset PYTHONPATH to ignore existing python installs.
 export PYTHONPATH=
-export CONDA_ENV=./wq_parsl
+export CONDA_ENV=./wq_vine_parsl
+export LOCAL_PARSL_SRC=./parsl-src
 
 trap cleanup EXIT
 
 cleanup() {
     rm -rf ${CONDA_ENV}
+    rm -rf ${LOCAL_PARSL_SRC}
+    rm -rf vine-run-info
```

Comment:
Any way we can make this the same as the parsl log directory?

Reply:
Do you mean the vine-run-info to be the same as runinfo of parsl?

Reply:
With wq, wq logs are written to same place that parsl sets. We need to do the same for taskvine.

Reply:
Got it. Just push the fix on my parsl fork and here. The solution was to add the log path parameter as a keyword argument instead of positional argument.